### ISSUE TITLE
viàATV

### ISSUE THREAD

**Issue by Europe57** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#349]
### Content description

* Name: viàATV
* TV channel/Catch-up
* Website URL: https://viaatv.tv/
* Content language/country: Français / France (Martinique, Guadeloupe, Guyanne)

### Content access

- [ ] This content or some part of this content require a free account
- [ ] This content or some part of this content require a paid account
- [ ] This content is geographically restricted

### More information

If you want to add more information, put it here:

Le live sur le site est coupe chaque jours, il ne marche que lors des JTs par contre les flux directs (m3u8) marche 24h/24.

Qualité "medium" : https://streamer03.myvideoplace.tv/streamer02/hls/ATV_DIRECT_EV_111018_medium/index.m3u8
Qualité "low" : https://streamer03.myvideoplace.tv/streamer02/hls/ATV_DIRECT_EV_111018_low/index.m3u8

Un mini soucis, l'audio est décaler de 0,750s (sous VLC)

**Comment by wwark** [comment]
Bonjour,
Ces flux vidéos ne sont pas ceux du site officiel donc nous ne pouvons pas les ajouter sur le plugin qui ne contient que des flux vidéos officiels et proposés directement par la chaine.
Sur le site officilel, on a bien le comportement que vous décrivez.
https://viaatv.tv/direct-tv/
Le plugin le reproduit.

Merci,
Wwark

**Comment by Europe57** [comment]
Bonjour,

D'accord par contre les 2 flux donnée sont bien les flux accessibles lors des directs sur leur site web, après je comprend que ça veut pas.

Par contre le replay peut être rajouter ?

Merci.

**Comment by wwark** [comment]
Oui ces deux liens renvois HTTP 403 tant que le live TV n'est pas disponible on ne peut rien faire sur ce point.
Le décalage vous l'avez aussi dans Kodi, VLC et le site web ?

On va mettre dans la TODO pour rajouter les replays.

Merci !

**Comment by Europe57** [comment]
Oui, le décalage est partout, Kodi, VLC, MPC-BE, MPC-HC et site web

Merci pour le replay.

**Comment by wwark** [comment]
Si le décalage est partout on ne peut rien y faire aussi ....

**Comment by Europe57** [comment]
Oui je me doute bien. Le replay y pas de soucis de son par contre, un bon point puisque le live ne peut pas.
